data/brands/amenity/car_rental.json

```
@@ -6,6 +6,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Green Motion Car Rental",
+      "locationSet": {"include": ["001"]},
+      "tags": {
+        "amenity": "car_rental",
+        "brand": "Green Motion Car Rental",
+        "brand:wikidata": "Q65315691",
+        "name": "Green Motion Car Rental"
```

```suggestion
      "displayName": "Green Motion",
      "locationSet": {"include": ["001"]},
      "matchNames": ["green motion car rental"],
      "tags": {
        "amenity": "car_rental",
        "brand": "Green Motion",
        "brand:wikidata": "Q65315691",
        "name": "Green Motion"
```
This seems to be how most existing locations are mapped.